packages/react-native-reanimated/apple/worklets/apple/WorkletsModule.mm

```diff
@@ -16,6 +17,9 @@ - (void)_tryAndHandleError:(dispatch_block_t)block;
 
 @implementation WorkletsModule {
   std::shared_ptr<NativeWorkletsModule> nativeWorkletsModule_;
+#ifndef NDEBUG
+  SingleInstanceChecker<WorkletsModule> singleInstanceChecker_;
```

Comment:
I think we should either change it to `worklets::SingleInstanceChecker` or add `using worklets` to avoid potential compilation errors

Reply:
Yeah I noticed this on CI, fixed [f47d857](https://github.com/software-mansion/react-native-reanimated/pull/6694/commits/f47d8576f9a01c09a3b304e21a40ae95da56e313)